### DISCUSSION TITLE
v1.26.0

### DISCUSSION THREAD

**Discussion by mikecao** [umami-software/umami#965] [General]
## Updates
- New languages: Lithuanian and Urdu! 🌟 
- Added active user count to Realtime page.
- Added environment variable `REMOVE_TRAILING_SLASH` which forces the removal of trailing slashes from all incoming urls.
- Added environment variable `TRACKER_SCRIPT_NAME` to allow you to name the tracker script to something instead of `umami.js`.
- Added environment variable `DISABLE_LOGIN` to allow disabling the webapp login page.
- Authentication for the app now uses bearer tokens instead of cookies.
- Language translation are now pre-built instead of generating at build time.
- CSS updates for mobile
- Update Next.js to `12.0.10`
- Updated Prisma to `3.9.2`

## Fixes
- Fixed issue with setting custom date range.
- Fixed check for `className` in tracking script.

A huge thanks to all the contributors on this release! 🙏 @0xflotus @leits @alrico88 @adilj13 @ym-project @kiprasmel @lxndrblz @nfn @TszHong0411 @rogerclotet 

<hr /><em>This discussion was created from the release <a href='https://github.com/mikecao/umami/releases/tag/v1.26.0'>v1.26.0</a>.</em>

**Comment by abarichello** [comment]
Isn't TRACKER_SCRIPT_NAME missing in the Dockerfile? I tried settings that env on my docker-compose and it isn't working

**Comment by mikecao** [comment]
@abarichello Looks like we need to expose all the potential env variables to docker. Would welcome a PR.

**Comment by Mokkapps** [comment]
This should be a major version as it contains a breaking change. My API access doesn't work anymore by using cookies.
